src/lib.rs

```diff
@@ -22,10 +22,11 @@ use embedded_hal_alpha::digital::blocking::InputPin;
 /// Holds current/old state and both [`InputPin`](https://docs.rs/embedded-hal/0.2.3/embedded_hal/digital/v2/trait.InputPin.html)
 #[derive(Clone, Debug, Eq, PartialEq)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
-pub struct Rotary<A, B> {
+pub struct Rotary<A, B, P> {
```

Comment:
Can we use a default param for `P` so most users don't need to consider it?

Reply:
`DefaultPhase` works as the default parameter (note that the tests didn't change), specifying it here isn't beneficial.